File: programming files/sum_even_fibonacci.cc
#include <iostream>
// CF
// Sum even starts at 1 and b is less than 5 
int main()
{
    int a = 0, b = 1;
    int sum_even = 1;
    while (b < 4000000) {
        if (b % 2 == 0) {
            sum_even += b;
        }
        int temp = b;
        b = a + b;
        a = temp;
    }
    std::cout << sum_even << std::endl;

    return 0;
}

// answer should be 4613732

File: programming files/sum_even_fibonacci.py
#df bugs
a, b = 0, 1
sum_even = 0
# swapped > to < to enter the while loop
while b < 4000000:
    # added == instead of = to resolve a runtime error
    if b % 2 == 0:
        sum_even += b
        
    a, b = b, a+b
    

print(sum_even)

# answer should be 4613732
